language/stdlib/modules/VASP.move

```diff
@@ -154,6 +146,9 @@ module VASP {
                 global<ChildVASP>(addr).parent_vasp_addr
             }
         }
+        define spec_has_account_limits<Token>(addr: address): bool {
```

Comment:
@emmazzz: I had to define a spec function for this because the prover wasn't happy with the purity of `has_account_limits`. Do you know why? Is it because it's not happy with the purity of `parent_address`?

Reply:
That's right. `parent_address` has an `abort` in it, which is currently not supported.

Reply:
Is there a conceptual blocker to treating funs with `abort`s as pure, or just an implementation one? I would guess the latter since @wrwg told me that spec functions can be partial, but wanted to make sure I understand.